packages/terra-i18n/tests/nightwatch/i18n-spec.js

```diff
@@ -12,6 +12,8 @@ const defaultTranslations = enTransaltions;
 
 const screenshot = require('terra-toolkit').screenshot;
 
+const pauseInms = 1000;
```

Comment:
Take a look at this to see what I mean: https://github.com/nightwatchjs/nightwatch/wiki/Understanding-the-Command-Queue